types/actions/omfwd.pp

```diff
@@ -0,0 +1,27 @@
+type Rsyslog::Actions::Omfwd = Struct[{
+  Target                            => Optional[Variant[Stdlib::Fqdn, Stdlib::IP::Address]],
```

Comment:
I saw that, but I wanted to stick with the documented "current" types rather than relying on something using a `compat` that seems to be there for backwards compat only